ios/FluentUI/List/ListItem.swift

```diff
@@ -0,0 +1,209 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+public typealias ListItemAccessoryType = TableViewCellAccessoryType
+public typealias ListItemBackgroundStyleType = TableViewCellBackgroundStyleType
+public typealias ListItemCustomViewSize = MSFTableViewCellCustomViewSize
+public typealias ListItemTokenSet = TableViewCellTokenSet
+
+/// View that represents an item in a List.
+public struct ListItem<LeadingView: View,
```

Comment:
LeadingContent / TrailingContent (also in the public facing param name) seems to be more platform native. See HStack header definition.